### ISSUE TITLE
"Start update process" screen shows every time a pod is deleted

### ISSUE THREAD

**Issue by BarryWilliams** [oben01/charts#61]
### Chart Name

Homarr

### Chart Version

homarr-1.2.5

### Bug Description

When a pod gets deleted (and a new pod is created) - Homarr keeps initially showing the "Start update process", which includes asking to set the admin user and password. This is rather annoying.

My dashboards still persist, which is great.

#### Recreation steps:
1. Deploy the chart
2. Do initial setup and add some dashboard creation
3. Delete the pod (`kubectl delete pod homarr-xx-xx -n homarr`)
4. Go to the URL again - it will go through "Start update process"
5. Delete the pod again - it will go through "Start update process" again
Note: dashboards persist

RKE 1.4.10
Longhorn for PV

### values.yaml

```yaml
ingress:
  enabled: true
  hosts:
    - host: launch.kube.local
      paths:
        - path: /
          pathType: Prefix
env:
  DEFAULT_COLOR_SCHEME: dark
persistence:
  - name: homarr-config
    # -- Enable this persistent storage
    enabled: true
    # -- Storage class name
    storageClassName: longhorn
    # -- Access mode
    accessMode: "ReadWriteOnce"
    # -- Storage size
    size: "50Mi"
    # -- Mount path inside the pod
    mountPath: "/app/data/configs"
  - name: homarr-database
    enabled: true
    storageClassName: longhorn
    accessMode: "ReadWriteOnce"
    size: "50Mi"
    mountPath: "/app/database"
  - name: homarr-icons
    enabled: true
    storageClassName: longhorn
    accessMode: "ReadWriteOnce"
    size: "50Mi"
    mountPath: "/app/public/icons"
```

### Relevant log output

```shell
Several back-to-back entries of the following:

ℹ Instance is using a database on the file system. Checking if file '/data/db.sqlite' is writable...
ℹ Database is writeable
```

**Comment by BarryWilliams** [comment]
Adding /data to it's own PV worked.

I added this PVC to my values.yaml:

```yaml
  - name: homarr-data
    enabled: true
    storageClassName: longhorn
    accessMode: "ReadWriteOnce"
    size: "50Mi"
    mountPath: "/data"
```

**Comment by BarryWilliams** [comment]
It looks like the `/data` location is set as an environment variable in the docker image.
https://github.com/ajnart/homarr/blob/cc240f4f87ceec8afd4b3079430c8d2bb054dad7/Dockerfile#L45

**Comment by BarryWilliams** [comment]
Updating the environment variables in the values file worked.

Mine looks like this now:
```yaml
env:
  DEFAULT_COLOR_SCHEME: dark
  DATABASE_URL: file:/app/database/db.sqlite
```

Perhaps this can be added to the templates

**Comment by oben01** [comment]
@BarryWilliams thank you for the pull request. I'll take a look next week

**Comment by oben01** [comment]
@BarryWilliams could u please follow instructions on this PR : https://github.com/oben01/charts/pull/41 to update documentation and chart metadata. Thank u

**Comment by oben01** [comment]
@BarryWilliams i just added a contribution guide https://github.com/oben01/charts/blob/main/.github/CONTRIBUTING.md could u please follow the instructions.

**Comment by oben01** [comment]
Fixed in version 1.3.5
